core/src/main/java/com/graphhopper/routing/util/parsers/OSMCyclewayParser.java

```diff
@@ -10,30 +10,30 @@
 import java.util.List;
 
 public class OSMCyclewayParser implements TagParser {
-  
+
```

Comment:
Noticing this file has a lot of pure indentation changes; can you modify your editor setup to align with whatever tabbing style GraphHopper is using here?